core/src/neighbourhood/NeighbourhoodClient.ts

```
@@ -1,279 +1,397 @@
+    return neighbourhoodSendBroadcastU;
+  }
 
-    dispatchSignal(perspectiveUUID:string, signal: any) {
-        const handlers = this.#signalHandlers.get(perspectiveUUID)
-        if (handlers) {
-            handlers.forEach(handler => handler(signal))
-        }
+  dispatchSignal(perspectiveUUID: string, signal: any) {
+    const handlers = this.#signalHandlers.get(perspectiveUUID);
+    if (handlers) {
+      handlers.forEach((handler) => handler(signal));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)